packages/desktop/components/popups/CreateDelegationPopup.svelte

```diff
@@ -99,8 +100,13 @@
             )
             transactionInfo.preparedTransactionError = undefined
         } catch (error) {
-            transactionInfo.preparedTransaction = undefined
-            transactionInfo.preparedTransactionError = error
+            if (error.message?.includes('Addresses')) {
+                addressError = error.message
+                handleError(error)
```

Comment:
```suggestion
```
imo this line should be removed, as it only shows the error inline, not with a notification :pray: 
![image](https://github.com/iotaledger/firefly/assets/72078237/06aed5ec-49db-44a4-8f07-cdb1ac95ca1d)

Reply:
Why would you want to show a notification ? What's wrong with an inline error?

Reply:
i would like not to show the notification, only the inline error

Reply:
Ah, I misunderstood.